.gitignore

```diff
@@ -61,6 +61,7 @@ src/qt/bitcoin-qt.includes
 *.orig
 *.pyc
 *.o
+*.o.*
```

Comment:
Why?

Reply:
I'm glad you ask 😄try running `make -j10` and in parallel run `git status`. Some editors will refresh showing these temporary files (mostly `*.o.tmp`).